### ISSUE TITLE
Connecting to a database with password

### ISSUE THREAD

**Issue by GeoffreyPlitt** [rethinkdb/horizon#664]
I'm using ReThinkDB provided by Compose.io.

The connection string they provide has a username/password, i.e.:
rethinkdb://admin:password@host:port

I haven't found a way to properly get this to work with Horizon-- please advise?

**Comment by GeoffreyPlitt** [comment]
Nevermind, duplicate of #507

**Comment by deontologician** [comment]
Yeah, we really really want to do this. It's coming after #588 is implemented!

**Comment by danielmewes** [comment]
@deontologician Did you comment on the right issue? This will already work with Horizon 2.0 right?

**Comment by deontologician** [comment]
Oh, I'm sorry, I was thinking of password auth. Totally glossed over the text of this issue :disappointed:
